app/views/allocations/_awaiting.html.erb

```diff
@@ -16,14 +14,14 @@
     </tr>
     </thead>
     <tbody class="govuk-table__body">
-    <% @prisoners.each_with_index  do |prisoner, i| %>
```

Comment:
Worth extracting to partial?

Reply:
This has changed to @ offenders_awaiting_allocation as it needed merging with master